codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -195,14 +195,26 @@ private FlowState connectionState() {
     }
 
     private static FlowState state(Http2Stream stream) {
-        checkNotNull(stream, "stream");
-        return stream.getProperty(FlowState.class);
+        if (stream instanceof FastPropertyAccess) {
```

Comment:
@Scottmitch I'm not sure that defining interfaces for `LocalFlowState` and `RemoteFlowState` buy us anything, since none of the data is needed outside of the flow controllers.  We might as well just store them as `Object` in the stream and provide template methods for access. WDYT?